gammapy/stats/variability.py

```diff
@@ -62,6 +64,62 @@ def compute_fvar(flux, flux_err, axis=0):
     return fvar, fvar_err
 
 
+def compute_fpp(flux, flux_err, axis=0):
+    r"""Calculate the point-to-point excess variance.
+
+    This method accesses the ``FLUX`` and ``FLUX_ERR`` columns
+    from the lightcurve data.
+
+    F_pp is a quantity strongly related to F_var, probing the variability
```

Comment:
What is F_var? ;)

Reply:
Thanks @bkhelifi, you are right that it may not be fully clear... Is there a standard way to refer to the docstring of another function?

Reply:
By making a reference to the function `~gammapy.xx.yy'. It should work (TBC)